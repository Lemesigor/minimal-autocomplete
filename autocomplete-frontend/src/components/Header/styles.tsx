import { css } from '@emotion/react';

export const headerStyle = css`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  padding-top: 5rem; 
`;

export const h2Style = css`
  font-size: 4.5rem;
    color: rgb(69, 168, 100);
`;