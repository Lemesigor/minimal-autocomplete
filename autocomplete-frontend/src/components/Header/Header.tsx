/** @jsxImportSource @emotion/react */

import {h2Style, headerStyle} from "./styles";
export const Header = () => {
    return (
        <header css={headerStyle}>
            <h2 css={h2Style}>Autocomplete</h2>
        </header>
    );
}