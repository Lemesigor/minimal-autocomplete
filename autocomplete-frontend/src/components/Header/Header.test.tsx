import {render} from "@testing-library/react";

import {Header} from "./Header";

describe("Header", () => {
    it("should render the header", () => {
        const {container} = render(<Header />);
        expect(container).toMatchSnapshot();
    });

    it("should have header text", () => {
        const {getByText} = render(<Header />);
        expect(getByText("Autocomplete")).toBeInTheDocument();
    })
});