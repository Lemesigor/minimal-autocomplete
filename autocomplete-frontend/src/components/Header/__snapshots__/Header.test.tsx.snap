// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the header 1`] = `
<div>
  <header
    class="css-1188xm2"
  >
    <h2
      class="css-rnq8sf"
    >
      Autocomplete
    </h2>
  </header>
</div>
`;
